src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -249,14 +708,20 @@ def map_keyvalue_to_featureflag(keyvalue, show_conditions=True):
     if filters and state == FeatureState.ON:
         state = FeatureState.CONDITIONAL
 
-    feature_flag = FeatureFlag(feature_name,
-                               keyvalue.key,
-                               keyvalue.label,
-                               state,
-                               feature_flag_value.description,
-                               conditions,
-                               keyvalue.locked,
-                               keyvalue.last_modified)
+    feature_flag = FeatureFlag(
+        name=feature_name,
```

Comment:
Should we rename this 'name' field to 'id'?

Reply:
Yes we could I see no issue with that. Although this is what we had before I am wondering if there was a reason why we had it as `Name`

Reply:
This is the FeatureFlag representation that is displayed to the user. The other FeatureFlagValue class has it as `id`

Reply:
ok it makes sense if this is for display. lets keep using "name"